TLM/TLM/UI/MainMenu/VehicleRestrictionsButton.cs

```diff
@@ -1,13 +1,27 @@
 ﻿namespace TrafficManager.UI.MainMenu {
+    using System.Collections.Generic;
     using TrafficManager.State;
+    using TrafficManager.U.Button;
 
-    public class VehicleRestrictionsButton : MenuToolModeButton {
+    public class VehicleRestrictionsButton : BaseMenuToolModeButton {
         protected override ToolMode ToolMode => ToolMode.VehicleRestrictions;
 
-        protected override ButtonFunction Function => ButtonFunction.VehicleRestrictions;
+        public override void SetupButtonSkin(List<string> atlasKeys) {
+            // Button backround (from BackgroundPrefix) is provided by MainMenuPanel.Start
+            this.Skin = new U.Button.ButtonSkin() {
+                                                      Prefix = "VehicleRestrictions",
+                                                      BackgroundPrefix = "RoundButton",
+                                                      BackgroundHovered = true,
+                                                      BackgroundActive = true,
+                                                      ForegroundActive = true,
+                                                  };
+            atlasKeys.AddRange(this.Skin.CreateAtlasKeysList());
```

Comment:
Ensure that keys added to the atlas are unique. Got a feeling that background is added every time.